backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Fresh_Account_Info_Report.sql

```diff
@@ -337,121 +100,481 @@ CREATE OR REPLACE FUNCTION report.fresh_account_info_report(
+                  (CASE
+                       WHEN
+                               (SELECT COUNT(0)
+                                FROM Fact_Acct fa2
+                                WHERE fa.ad_table_id = fa2.ad_table_id
+                                  AND fa2.Record_ID = fa.Record_ID
+                                  AND fa.Fact_Acct_id != fa2.Fact_Acct_id
+                                  AND (CASE WHEN fa.amtacctdr != 0 THEN fa2.amtacctcr != 0 WHEN fa.amtacctcr != 0 THEN fa2.amtacctdr != 0 ELSE FALSE END)
+                                  AND fa2.isActive = 'Y')
+                               = 1 THEN (SELECT ev2.value || ' ' || ev2.name
+                                         FROM Fact_Acct fa2
+                                                  INNER JOIN C_ElementValue ev2 ON fa2.Account_ID = ev2.C_ElementValue_ID AND ev2.isActive = 'Y'
+                                         WHERE fa.ad_table_id = fa2.ad_table_id
+                                           AND fa2.Record_ID = fa.Record_ID
+                                           AND fa.Fact_Acct_id != fa2.Fact_Acct_id
+                                           AND (CASE WHEN fa.amtacctdr != 0 THEN fa2.amtacctcr != 0 WHEN fa.amtacctcr != 0 THEN fa2.amtacctdr != 0 ELSE FALSE END)
+                                           AND fa2.isActive = 'Y')
+                                   ELSE ''
+                   END)                                                                                                                                       AS Account2_ID, -- this selects the name and value of one or no element value, that is matching with the current fact_acct (see when you press verbucht on your docu there is more than 1 line). Later shall be changed in some way so it can selece more, but currently we cannot associate more
+
+                  CAST ( a.value AS text )                                                                                                                                     AS A_Value,
+                  fa.amtsourcedr,
+                  fa.amtsourcecr,
+                  fa.AmtAcctDr,
+                  fa.AmtAcctCr,
+                  SUM(fa.AmtAcctDr) OVER (PARTITION BY fa.param_acct_value)                                                                                      AS AmtAcctDrEnd,
+                  SUM(fa.AmtAcctCr) OVER (PARTITION BY fa.param_acct_value)                                                                                      AS AmtAcctCrEnd,
+                  fa.CarryBalance + SUM(Balance) OVER (
+                      PARTITION BY fa.Account_ID
+                      ORDER BY fa.Account_ID, fa.DateAcct, fa.Fact_Acct_ID
+                      )                                                                                                                                       AS saldo,
+                  fa.CarryBalance AS carrysaldo,
```

Comment:
imho u can rename CarryBalance (and carrysaldo) to `begining_balance`.

the `saldo` (which is beginning_balance + current balance) shall be called rolling_balance (i.e. the balance after current Fact_Acct line).



PS: also pls avoid mixing german names with english names (i know the reports are full of that but....)

Reply:
Because of limited time I didn‘t adjust old namings